src/js/grid.js

```diff
@@ -49,7 +54,9 @@ Grid.prototype = {
     },
 
     _html: function () {
-        var html = '<div class="medium-editor-table-builder-grid clearfix">';
+        var width = this.columns * 16 + 2,
```

Comment:
Where does the 16 come from?  If it doesn't make sense to call this out as a constant/option now, could you add a comment to explain where the number comes from and what it's purpose is?